win32/tests/gballoc_hl_metrics_wout_init_ut/gballoc_hl_metrics_wout_init_ut.c

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#ifdef __cplusplus
+#include <cstddef>
+#else
+#include <stddef.h>
+#endif
+
+#include "azure_macro_utils/macro_utils.h"
+#include "testrunnerswitcher.h"
+
+#include "umock_c/umock_c.h"
+#include "umock_c/umocktypes_stdint.h"
+#include "umock_c/umocktypes.h"
+#include "umock_c/umock_c_negative_tests.h"
+
+#include "azure_c_pal/gballoc_hl.h"
+
+#if defined(_MSC_VER)
+#pragma warning(disable : 4189)
+#endif
+
+#define ENABLE_MOCKS
+#include "azure_c_pal/gballoc_ll.h"
+#undef ENABLE_MOCKS
+
+static TEST_MUTEX_HANDLE test_serialize_mutex;
+
+MU_DEFINE_ENUM_STRINGS(UMOCK_C_ERROR_CODE, UMOCK_C_ERROR_CODE_VALUES)
+
+static void on_umock_c_error(UMOCK_C_ERROR_CODE error_code)
+{
+    ASSERT_FAIL("umock_c reported error :%" PRI_MU_ENUM "", MU_ENUM_VALUE(UMOCK_C_ERROR_CODE, error_code));
+}
+
+BEGIN_TEST_SUITE(gballoc_hl_metrics_wout_init_unittests)
+
+TEST_SUITE_INITIALIZE(suite_init)
+{
+    test_serialize_mutex = TEST_MUTEX_CREATE();
+    ASSERT_IS_NOT_NULL(test_serialize_mutex);
+
+    ASSERT_ARE_EQUAL(int, 0, umock_c_init(on_umock_c_error), "umock_c_init");
+
+    ASSERT_ARE_EQUAL(int, 0, umocktypes_stdint_register_types(), "umocktypes_stdint_register_types");
+}
+
+TEST_SUITE_CLEANUP(suite_cleanup)
+{
+    umock_c_deinit();
+
+    TEST_MUTEX_DESTROY(test_serialize_mutex);
+}
+
+TEST_FUNCTION_INITIALIZE(method_init)
+{
+    int result;
+
+    if (TEST_MUTEX_ACQUIRE(test_serialize_mutex))
+    {
+        ASSERT_FAIL("Could not acquire test serialization mutex.");
+    }
+
+    umock_c_reset_all_calls();
+    result = umock_c_negative_tests_init();

```

Comment:
>result = umock_c_negative_tests_init(); [](start = 4, length = 39)

Nitpick: consider not having result and directly asserting.

Reply:
I do remember changing all these SOME TIME IN THE PAST. How did it creep back... 

Not used, so removed completely anyway. Here and everywhere

---
In reply to: [459663283](https://github.com/Azure/azure-c-pal/pull/14#discussion_r459663283) [](ancestors = 459663283)